File: Friends.NET/Friends.Common/Application/Models/FriendUI.cs
using System;
using CommunityToolkit.Mvvm.ComponentModel;

namespace Friends.Common.Application.Models
{
	public partial class FriendUI : ObservableObject
	{
        public string FirstName { get; set; } = "";
        public string LastName { get; set; } = "";
        public string NickName { get; set; } = "";
        public string ProfilePictureUrl { get; set; } = "";
        [ObservableProperty]
        public bool isFriend;
        public int MajorsWon { get; set; }

        public string Id { get; set; } = "";
        public string FullName { get; set; } = "";
        public string Age { get; set; } = "";
    }
}

File: Friends.Kotlin/app/src/main/java/com/friends/presentation/models/FriendUi.kt
package com.friends.presentation.models

import java.time.LocalDate

data class FriendUi(
    val id: String,
    val firstName: String,
    val lastName: String,
    val isFriend: Boolean,
    val nickName: String,
    val birthDate: LocalDate,
    val age: String,
    val fullName: String
)
